tool/autoqa/normalize-data.js

```diff
@@ -526,7 +526,7 @@ class Normalizer {
 
 module.exports = {
     initArgparse(subparsers) {
-        const parser = subparsers.addParser('webqa-normalize-data', {
+        const parser = subparsers.addParser('autoqa-normalize-data', {
```

Comment:
This one is schema.org specific right? Unlike the other ones. Perhaps this should be reflected in the command name.